src/server/auth/cmds/cmds.go

```diff
@@ -291,14 +286,14 @@ func CheckCmd(noMetrics, noPortForwarding *bool) *cobra.Command {
 			return nil
 		}),
 	}
-	return check
+	return cmdutil.CreateAliases(check, []string{"auth check"})
 }
 
 // GetCmd returns a cobra command that gets either the ACL for a Pachyderm
 // repo or another user's scope of access to that repo
-func GetCmd(noMetrics, noPortForwarding *bool) *cobra.Command {
-	setScope := &cobra.Command{
-		Use:   "get [username] repo",
+func GetCmd(noMetrics, noPortForwarding *bool) []*cobra.Command {
+	get := &cobra.Command{
+		Use:   "{{alias}} [<username>] <repo>",
```

Comment:
That would be nice, but I don't think we have any option but to break compatibility to make that change.  I believe it would be subtle, based on cobra's routing, and it would only behave differently if the user had a username or repo called `acl`.  I'll add a top-level comment about other renames.